.nais/alerts.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: "nais.io/v1"
+kind: "Alert"
+metadata:
+  name: isdialogmelding-alerts
+  namespace: teamsykefravr
+  labels:
+    team: teamsykefravr
+spec:
+  receivers:
+    slack:
+      channel: '#syfo-alarm'
+      prependText: '<!here> | '
+  alerts:
+    - alert: ISDIALOGMELDING IS DOWN
+      expr: sum(up{app="isdialogmelding", job="kubernetes-pods"}) == 0
+      for: 5m
+      description: "Det er ingen pods for app kjørende i namespace og dermed er appen nede"
+      action: "Undersøk hvorfor pods er nede og få den kjørende igjen!"
+      severity: danger
+    - alert: FAILURES IN CRONJOB FOR SENDING DIALOGMELDING
+      expr: sum(increase(isdialogmelding_cronjob_dialogmelding_fail_count_total{app="isdialogmelding"} [1h])) > 0
```

Comment:
Denne ville ikke ha fanget opp situasjonen vi hadde i går, men jeg tror det ikke er så enkelt.

Reply:
Kunne kanskje lagt inn en visning antall success og feil i Grafana? Så aner man kanskje noe rusk med jobben hvis alt er 0 der. 
Eller om vi har en metrikk på inkommende dialogmelding-bestillinger som man kunne sammenlignet med antall sendte?

Reply:
Man kan jo lage alerts i Grafana også, så man kan få varsel hvis forholdet mellom bestillinger og utsendinger blir for lavt.

Reply:
Gode ideer - skal se litt mer på det.

Reply:
Ser vi har noe ubrukt kode på innkommende dialogmelding-bestilling-metrikker: https://github.com/navikt/isdialogmelding/blob/master/src/main/kotlin/no/nav/syfo/behandler/kafka/KafkaMetric.kt